pandas/core/frame.py

```diff
@@ -217,6 +217,7 @@ def __init__(self, data=None, index=None, columns=None, dtype=None,
             dtype = self._validate_dtype(dtype)
 
         if isinstance(data, DataFrame):
+            data = data.iloc[:,:]
             data = data._data
```

Comment:
@jreback OK -- best way to get a view at the level of BlockManager object? I understand how to get them at the level of DataFrames (`df.iloc[:,:]` works safely, but I know it's a little hacky) , but don't know how best to get a view reliably.